webservices/decoders.py

```diff
@@ -15,11 +15,12 @@
 
 form_types = {
     'F1': 'Statement of organization',
-    'F1M': 'Notification Of Multicandidate Status',
+    'F1M': 'Notification of multicandidate Status',
     'F2': 'Statement of candidacy',
     'F9': '24-hour notice of disbursements for electioneering communications',
     'F10': '24-hour notice of expenditure of personal funds',
     'F11': '24-hour notice of opposition personal funds amount',
     'F12': '24-hour notice of suspension of increased limits',
-    'F99': 'Miscellaneous Document',
+    'F99': 'Miscellaneous document',
+    'F6': '48-Hour notice contribution or loans',
```

Comment:
For consistency, how about lower-casing "Hour", and using the full title of the form: "48-hour notice of contributions/loans received"?